benchmark/src/main/java/org/eclipse/rdf4j/benchmark/FastRdfsForwardChainingSailBenchmark.java

```diff
@@ -0,0 +1,34 @@
+/*******************************************************************************
+ * Copyright (c) 2016 Eclipse RDF4J contributors.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Distribution License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/org/documents/edl-v10.php.
+ *******************************************************************************/
+
+
+package org.eclipse.rdf4j.benchmark;
+
+import org.eclipse.rdf4j.repository.sail.SailRepository;
+import org.eclipse.rdf4j.sail.inferencer.fc.FastRdfsForwardChainingSail;
+import org.eclipse.rdf4j.sail.memory.MemoryStore;
+
+/**
+ * @author Håvard Mikkelsen Ottestad
+ */
+public class FastRdfsForwardChainingSailBenchmark extends RdfsBenchmark {
+
+
```

Comment:
We have eclipse formatting settings, you can find them in `/eclipse-settings/rdf4j-formatting.xml`. If I understand this [StackOverflow question](http://stackoverflow.com/questions/27035705/how-import-eclipse-code-style-to-intellij-idea) correctly, it's possible to import such a settings file into IntelliJ. It will not map 100% identical but should set most settings the same or similar. Give it a try, see how you fare.